values-cs/strings.xml

```diff
@@ -2,126 +2,126 @@
+    <string name="time">Čas</string>
+    <string name="sign_in_with_google_dialog_message">Přihlaš se přes google k udělování úspěchů a synchronizací vašeho nejvyššího skóre do cloudu.</string>
+    
     <!-- SETTINGS ACTIVITY -->
-    <string name="about">About</string>
-    <string name="rate">Rate</string>
-    <string name="share">Share</string>
-    <string name="show_us_your_support">Show us your support!</string>
-    <string name="share_with_your_friends">Share 2048++ with your friends!</string>
-    <string name="google_play_games">Google Play Games</string>
-    <string name="sign_in">Sign in</string>
-    <string name="sign_out">Sign out</string>
-    <string name="please_sign_in_again">Please sign in again</string>
-    <string name="sign_in_with_google_play_games">Sign in with Google Play Games</string>
-    <string name="manage_your_google_play_games_settings">Manage your Google Play Games settings</string>
+    <string name="about">O nás</string>
+    <string name="rate">Ohodnotit</string>
+    <string name="share">Sdílet</string>
+    <string name="show_us_your_support">Ukaž nám svou podporu!</string>
+    <string name="share_with_your_friends">Sdílej 2048++ s přáteli!</string>
+    <string name="google_play_games">Google Play Hry</string>
+    <string name="sign_in">Přihlásit se</string>
+    <string name="sign_out">Odhlásit se</string>
+    <string name="please_sign_in_again">Prosím přihlaš se znovu</string>
+    <string name="sign_in_with_google_play_games">Přihlaš se přes Google Play hry</string>
+    <string name="manage_your_google_play_games_settings">Spravuj nastavení google play her</string>
     <!-- The format string argument is an email address (i.e. "You are about to sign out of this game as alexjlockwood@gmail.com.") -->
-    <string name="confirm_sign_out_message">You are about to sign out of this game as %1$s.</string>
+    <string name="confirm_sign_out_message">Opravdu se chceš odhlásit ze hry jako %1$s.</string>
     <!-- The format string argument is an email address (i.e. "Currently signed in as: alexjlockwood@gmail.com") -->
-    <string name="currently_signed_in_as">Currently signed in as: %1$s</string>
+    <string name="currently_signed_in_as">Právě přihlášen jako: %1$s</string>
 
     <!-- ABOUT ACTIVITY -->
-    <string name="contact_us">Contact Us</string>
+    <string name="contact_us">Kontaktuj nás</string>
     <string name="developed_and_designed_by">Developed &amp; Designed By</string>
-    <string name="special_thanks_to">Special Thanks To</string>
+    <string name="special_thanks_to">Speciální poděkování</string>
     <!-- This is where I will give you credit in the About activity for helping me translate the app. :)
          This string isn't used in the application yet... but it will be when the translations are released! -->
-    <string name="translations_by">Translations By</string>
+    <string name="translations_by">Překlad:</string>
 
     <!-- HOW TO PLAY ACTIVITY -->
```

Comment:
For these strings, can you also add the <b>...</b> tags as they are used in the English translation? Some words are bold to add emphasis here. :)